### PR TITLE
12751 Added Business BCeID card

### PR DESCRIPTION
**Pull request by severinbeauvais**
Issue: https://github.com/bcgov/entity/issues/12751

Changes:
- added BCeID URL to example env file
- added BCeID URL to cloudbuild scripts
- added BCeID URL to NUXT config file
- updated app version to 1.0.23
- updated caniuse package
- added new card for Business BCeID
- added keys to cloudbuild scripts per Doug
- updated section title per Yui

### CODE REVIEW

**Comment by severinbeauvais** [comment]
@thorwolpert @pwei1018 @doug-lovett Could one of you unblock gcbrun for me, please? I ran it an hour ago and it's still queued.

**Comment by severinbeauvais** [comment]
@doug-lovett @thorwolpert Can one of you add the new environment variable, please?

**Comment by doug-lovett** [comment]
@severinbeauvais @thorwolpert New env var added to the gcb trigger script. Before rerunning gcbrun, can you please add the following in this PR to cloudbuild-cd.yaml and cloudbuild-pr.yaml (missed in the last PR).
- 'BCA_URL=$_BCA_URL'
- 'BCA_LEARN_MORE_URL=$_BCA_LEARN_MORE_URL'
- 'ESRA_URL=$_ESRA_URL'
- 'ESRA_LEARN_MORE_URL=$_ESRA_LEARN_MORE_URL'
- 'CSO_URL=$_CSO_URL'
- 'CSO_LEARN_MORE_URL=$_CSO_LEARN_MORE_URL'

Thanks

**Comment by severinbeauvais** [comment]
> @severinbeauvais @thorwolpert New env var added to the gcb trigger script. Before rerunning gcbrun, can you please add the following in this PR to cloudbuild-cd.yaml and cloudbuild-pr.yaml (missed in the last PR).

Thanks. Done!

**Review by seeker25 — COMMENTED** [review]
I can't speak much to this repo, I don't have much experience deploying it

**Review by pwei1018 — APPROVED** [review]

**Review by doug-lovett — APPROVED** [review]

**Comment by severinbeauvais** [comment]
Can this PR be merged and deployed now?

File: cloudbuild-pr.yaml [hunk 1]
```diff
@@ -9,27 +9,34 @@ steps:
...
-    - 'BUSINESS_URL=$_BUSINESS_URL'
-    - 'BCREGISTRY_LD_CLIENT_ID=$_BCREGISTRY_LD_CLIENT_ID'
     - 'RPT_URL=$_RPT_URL'
-    - 'BUSINESS_SEARCH_URL=$_BUSINESS_SEARCH_URL'
-    - 'BUS_SEARCH_LEARN_MORE=$_BUS_SEARCH_LEARN_MORE'
     - 'SITEMINDER_LOGOUT_URL=$_SITEMINDER_LOGOUT_URL'
+    - 'SOCIETIES_URL=$_SOCIETIES_URL'
+    - 'STATUS_API_URL=$_STATUS_API_URL'
+    - 'WILLS_LEARN_MORE_URL=$_WILLS_LEARN_MORE_URL'
+    - 'WILLS_URL=$_WILLS_URL'
```
Comment (severinbeauvais) [inline cloudbuild-pr.yaml:39]: I sorted the keys, same as in the CD script.

File: .env.example [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
-BCA_URL = "https://bcrosweb-dev.bcassessment.ca:54579/"
-BCA_LEARN_MORE_URL = ""
-ESRA_URL = "https://esra-dlvr-frontend.apps.silver.devops.gov.bc.ca/"
-ESRA_LEARN_MORE_URL = "https://esra-prd-frontend.apps.silver.devops.gov.bc.ca/"
-CSO_URL = "https://dev.justice.gov.bc.ca/cso/index.do"
-CSO_LEARN_MORE_URL = "https://justice.gov.bc.ca/cso/index.do"
+SOCIETIES_URL = "https://dev.bcregistry.ca/societies/"
+STATUS_API_URL = "https://status-api-dev.apps.silver.devops.gov.bc.ca/api/v1/"
+WILLS_URL = "https://ewills-dev.vs.gov.bc.ca/"
+WILLS_LEARN_MORE_URL = "https://www2.gov.bc.ca/gov/content/life-events/death/wills-registry"
```
Comment (severinbeauvais) [inline .env.example:28]: I sorted this, too.

The new file has 1 new key (BCEID_URL).

**Pull request merged**